tools/linter_lib/printer.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import print_function, absolute_import
+from itertools import cycle
+
+from typing import Union, Text
+
+colors = cycle(['\x1B[32m', '\x1B[33m', '\x1B[34m', '\x1B[35m', '\x1B[36m'])
+
+
+def print_err(name, color, line):
+    # type: (str, str, Text) -> None
+    """
+    '\x1B[1;31m': Bold red
+    '\x1B[0m': Reset color
+    """
+    print('{}{}{}|\x1B[0m \x1B[1;31m{}\x1B[0m'.format(
```

Comment:
`scripts/lib/zulip_tools.py` has these colors in variables, e.g. `FAIL` and `ENDC`.  We should probably either use those colors, or, since this might be a library we extract from the rest of the codebase, at least give them human-readable names.